server_admin/topics/users/proc-setting-default-required-actions.adoc

```diff
@@ -0,0 +1,16 @@
+[id="proc-setting-default-required-actions_{context}"]
+= Default Required Actions
+
+You can also specify required actions that will be added to an account in two ways.  
```

Comment:
How about:  "You can use two methods to specify required actions that will be added to an account."